### PR TITLE
Make APP_URL configurable via helm templating

### PR DESCRIPTION
**Pull request by c0d1ngm0nk3y**
Purpose: To integrate with service manager not hard coded via extension, but loosely coupled via chaining.

### CODE REVIEW

**Comment by c0d1ngm0nk3y** [comment]
@dzahariev How to assign/get reviewers?

File: charts/service-broker-proxy-k8s/values.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ image:
 service:
   type: ClusterIP
   port: 80
+  app_url:
```
Comment (c0d1ngm0nk3y) [inline charts/service-broker-proxy-k8s/values.yaml]: We haven't documented the parameter since all service parameter are undocumented.

**Comment by c0d1ngm0nk3y** [comment]
It looks to me that travis is always failing when we do small PR to this repo :( But always unrelated...

**Comment by dzahariev** [comment]
The Travis job is actually pretty stable.
The failure was related to outage in coveralls.io service.

**Comment by c0d1ngm0nk3y** [comment]
What is the state of this PR? It is rather small, isn't it?

**Comment by dzahariev** [comment]
Waits for review

**Pull request merged**